resources/views/checkout/partials/address-cards-popup.blade.php

```diff
@@ -0,0 +1,30 @@
+<div
+    class="fixed inset-0 opacity-0 transition z-50 flex justify-center items-center"
+    :class="{ 'opacity-100': popup, 'pointer-events-none': !popup }"
+>
+    <div class="absolute inset-0 bg-primary/60" v-on:click="togglePopup"></div>
+    <div class="bg-inactive-100 p-3 rounded z-10 relative">
+        <x-rapidez-ct::title class="text-24 p-3">@lang('My addresses')</x-rapidez-ct::title>
+        <x-heroicon-o-x v-on:click="togglePopup" class="absolute top-7 right-7 w-5 h-5 cursor-pointer"></x-heroicon-o-x>
+
+        <div class="grid grid-cols-2 gap-5 p-3">
+            <x-rapidez-ct::card.address
+                v-for="userAddress in $root.user.addresses"
+                v-bind:key="userAddress.id"
+                v-bind:address="userAddress"
+                v-bind:billing="isType('billing', userAddress)"
+                v-bind:shipping="isType('shipping', userAddress)"
+                dynamic-type
+                check="isType('billing', userAddress) || isType('shipping', userAddress)"
+                class="min-w-[350px]"
```

Comment:
why the min-width? this probably is going to stick out of your screen on mobiles which have 320px resolutions, ask me in person to fix this together